exporters/committime/app.py

```diff
@@ -143,7 +147,7 @@ def make_collector(self) -> AbstractCommitCollector:
                 api = {}
             return GitHubCommitCollector(
                 kube_client=self.kube_client,
-                username=self.username,
+                # username=self.username,
```

Comment:
should we remove this since it is not necessary?

Reply:
I will remove all the code changes and talk to Kevin where is best to introduce them